src/service/chat.js

```diff
@@ -191,7 +193,8 @@ const moveChatToAnon = async ({
       anon_user_info_emoji_name,
       oldChannelId,
       targetUserId,
-      source
+      source,
+      context: 'chat'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `context` field is hardcoded as 'chat'. If this value is supposed to vary based on different scenarios, consider making it a parameter instead of hardcoding it.

```diff
-       context: 'chat'
+       context: context || 'chat'
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->